README.md

```diff
@@ -0,0 +1,22 @@
+# Additional Fruits
+This module adds various different fruit blocks and includes 3 different stages for each plant.
+
+**Types of Fruits:**
```

Comment:
IMHO this list is too much of a transcription of what's currently in the module, right down to the directory structure. For instance Blueberry should probably actually be under the "Bushes" dir but just got placed one level higher coincidentally. Again an example of being overly verbose without really telling the reader anything useful - it doesn't matter that some of the bushes are nested under a subdir.

Instead you may want to focus on the sheer fact that there *are* bushes stored here, which in turn is one system of fruits with associated code (but where? Now *that* I bet the reader would love to know!). Then in the future as other fruit types are added they could each get their own bullet point or section.